variables.tf

```diff
@@ -0,0 +1,77 @@
+
+variable "description" {
+  default     = ""
+  description = "The description of the key as viewed in AWS console."
+  type        = string
+}
+
```

Comment:
For the variables with default being `" "` are they required? If not so, can they be defaulted to `null`